internal/printer/print-to-js.go

```diff
@@ -117,13 +116,17 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 				p.addSourceMapping(n.Loc[0])
 				if renderBodyStart == -1 {
 					p.addSourceMapping(c.Loc[0])
-					if js_scanner.AccessesPrivateVars([]byte(c.Data)) {
-						panic(errors.New("Variables prefixed by \"$$\" are reserved for Astro's internal usage!"))
-					}
```

Comment:
I like this change. Does the compiler now resolve conflicts differently?

Reply:
Nope, this warning never worked properly. I'd like to automatically namespace things so this isn't even necessary (_eventually_)